grpc-api/src/main/java/io/deephaven/grpc_api/table/TableServiceGrpcImpl.java

```diff
@@ -1,6 +1,6 @@
 package io.deephaven.grpc_api.table;
 
-import io.deephaven.io.logger.Logger;
+import com.google.flatbuffers.FlatBufferBuilder;
```

Comment:
Accidental import?

Reply:
No, reordered to be alphabetical, grouped by prefix (`com.*` with other `com.*`)